ntoskrnl/mm/ARM3/sysldr.c

```diff
@@ -1590,7 +1592,12 @@ MiFindInitializationCode(OUT PVOID *StartVa,
                     else
                     {
                         /* This isn't us -- go ahead and free it */
-                        ASSERT(MI_IS_PHYSICAL_ADDRESS((PVOID)InitStart) == FALSE);
```

Comment:
Why removing this assert?

Reply:
Maybe because the name MI_IS_PHYSICAL_ADDRESS always make me cringe...
Anyway, I added it back.